llama_stack/apis/agents/agents.py

```diff
@@ -364,7 +362,7 @@ class AgentTurnCreateRequest(AgentConfigOverridablePerTurn):
     ]
 
     documents: Optional[List[Document]] = None
-    toolgroups: Optional[List[AgentToolGroup]] = None
+    toolgroups: list[AgentToolGroup] | None = Field(default_factory=lambda: [])
```

Comment:
nit: List vs list

Reply:
Sorry what nit? Using list[...] is the modern syntax valid in Python 3.9+. Since we’re using Python 3.10+ it's all good.

Reply:
i agree.

we're inconsistently using list vs List across the project. iirc, at the time this was mixing within the file.

Reply:
Ahhh I see but now of that is fixed with https://github.com/meta-llama/llama-stack/pull/1806 :)